google-datacatalog-connectors-commons/src/google/datacatalog_connectors/commons/utils/region_tag_helper.py

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/python
+#
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import logging
+import re
+
+
+class RegionTagHelper:
+    """
+    Regex pattern to split string into 3 groups:
+
+    Group 1: [{REGION_TAG_NAME}_START]
+    Group 2: Content between START and END region tags
+    Group 3: [{REGION_TAG_NAME}_END]
+    """
+    __REGION_TAG_GROUP_REGEX = re.compile(
+        r"^(\s*\[\S*_START\][^\S\r\n]*)((?s:.)*)(\s*\[\S*_END\]\s*)",
```

Comment:
```suggestion
        r'^(\s*\[\S*_START\][^\S\r\n]*)((?s:.)*)(\s*\[\S*_END\]\s*)',
```
Do single quotes work here? If so, I believe it's better to keep consistency.

Also, I'd rather go with named groups. They improve the regex readability. Did you give them a try?